### PR TITLE
SOC-1035 New JA translation for login and registration

### PR DESCRIPTION
**Pull request by rafalkalinski**
https://wikia-inc.atlassian.net/browse/SOC-1035

### CODE REVIEW

File: server/facets/auth/register.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export function get (request: Hapi.Request, reply: any): Hapi.Response {
 			headerText: 'auth:join.sign-up-with-email',
 			heliosRegistrationURL: localSettings.helios.host + '/users',
 			title: 'auth:join.sign-up-with-email',
-			termsOfUseLink: 'http://www.wikia.com/Terms_of_Use',
+			termsOfUseLink: '<a href="' + i18n.t('auth:register.terms-of-use-link-url') + '" target="_blank">' + i18n.t('auth:register.terms-of-use-link-title') + '</a>',
```
Comment (bkoval) [inline server/facets/auth/register.ts]: How about moving it to AuthUtils with a method like "getTermsOfUseMarkup" or sth? :)

File: front/locales/en/auth.json [hunk 2]
```diff
@@ -30,8 +30,9 @@
 		"footer-callout-link": "Log in",
 		"register-action": "Register",
 		"page-title": "Register with Email",
-		"terms-of-use-text": "By creating an account, you agree to",
-		"terms-of-use-link": "Wikia's Terms of use",
+		"terms-of-use-text": "By creating an account, you agree to __termsOfUseLink__",
+		"terms-of-use-link-title": "Wikia's Terms of use",
+		"terms-of-use-link-url": "http://www.wikia.com/Terms_of_Use",
```
Comment (kvas-damian) [inline front/locales/en/auth.json]: I don't like storing URLs in i18n files. What do you think about moving it to ```...Settings.ts``` file?

File: server/facets/auth/register.ts (hunk 1 above)
Comment (kvas-damian) [inline server/facets/auth/register.ts]: Can't it be done in hbs template?

**Comment by kvas-damian** [comment]
LGTM

File: front/locales/en/auth.json [hunk 3]
```diff
@@ -30,7 +30,7 @@
 		"footer-callout-link": "Log in",
 		"register-action": "Register",
 		"page-title": "Register with Email",
-		"terms-of-use-text": "By creating an account, you agree to",
+		"terms-of-use-text": "By creating an account, you agree to __termsOfUseLink__",
```
Comment (jsutterfield) [inline front/locales/en/auth.json]: Is `__termsOfUseLink__` a variable which gets interpolated later? None of the other languages seem to have it. Or is it something else? The leading and trailing double underbars make me think it's some kind of special symbol.

Comment (rafalkalinski) [inline front/locales/en/auth.json]: It is a variable. It needs to be in the other languages, but as far I don't know them, I'm not sure it the variable should be placed at the end of the sentence - it is up to translators.

**Pull request merged**